src/platform/plugins/shared/discover/public/application/main/components/layout/cascaded_documents/index.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+export { CascadedDocumentsLayout, useGroupBySelectorRenderer } from './cascaded_document_layout';
```

Comment:
```suggestion
export { useGroupBySelectorRenderer } from './cascaded_document_layout';
```
Will `LazyCascadedDocumentsLayout` async import as intended if both are exported here, and `useGroupBySelectorRenderer` is also imported in `discover_grid.ts`? Either way it looks like this export isn't used.